scripts/etc/udhcpc/default.script

```diff
@@ -19,7 +19,7 @@ PEERDNS_IF=eth0
 case "$1" in
   deconfig)
     # bring interface up, but with no IP configured:
-    ip addr flush dev $interface
+    ip -f inet addr flush dev $interface
     ip link set $interface up
```

Comment:
because the link command affects the network device, so protocol families don't make sense here...